WalletWasabi/Legal/Assets/LegalDocuments.txt

```diff
@@ -64,7 +64,7 @@ I. TERMS AND CONDITIONS
 2.1 DESCRIPTION OF SERVICES
     This Software consists of the Client Application and the CoinJoin Coordinator (both as defined below) and functions as a free, open-source, non-custodial desktop Bitcoin wallet. The Software does not constitute an account where we or other third parties serve as financial intermediaries or custodians of Your bitcoins. The Software and our Services are therefore not a bank, a custodian, an exchange, a financial intermediary or a regulated financial institution and is exempt from the authority of the Gibraltar Financial Services Commission or any other financial institutions.
 
-    The client application (“Client Application”) is software, with the sole purpose of allowing You access to the Bitcoin network and our Services (as defined below), without the need or requirement to create or maintain a user account. The application itself is an ordinary Bitcoin desktop wallet with ordinary features. Wasabi Wallet does not store, send or receive bitcoins. This is because bitcoins exist only by virtue of the ownership record maintained in the Bitcoin network. Any transfer of title in bitcoins occurs within the decentralized Bitcoin network, and not in the Wasabi Wallet.
+    The client application (“Client Application”) is software, with the sole purpose of allowing You to access to the Bitcoin network and our Services (as defined below), without the need or requirement to create or maintain a user account. The application itself is an ordinary Bitcoin desktop wallet with ordinary features. Wasabi Wallet does not store, send or receive bitcoins. This is because bitcoins exist only by virtue of the ownership record maintained in the Bitcoin network. Any transfer of title in bitcoins occurs within the decentralized Bitcoin network, and not in the Wasabi Wallet.
```

Comment:
@bharmat pls approve.

Reply:
TBH, I'm not sure it's a typo. To me, it looks good without `to`.

Reply:
I agree with @nopara73. It is fine without to.

Reply:
Let's not touch it, to stay safe.